website/source/guides/upgrade/upgrade-specific.html.md

```diff
@@ -15,6 +15,32 @@ details provided for their upgrades as a result of new features or changed
 behavior. This page is used to document those details separately from the
 standard upgrade flow.
 
+## Nomad 0.10.0
+
+### Deployments
+
+Nomad 0.10 enables rolling deployments for service jobs by default
+and adds a default update stanza when a service job is created or updated.
+This does not affect jobs with an update stanza.
+In pre-0.10.0 releases, service jobs without an update stanza were deployed
+with destructive updates by default. To regain this behavior and
```

Comment:
Not sure if we ever called it "destructive updates" before.  If not, we might tweak the language to explain behavior.  Something like:

> In pre-0.10 releases, when updating a a service job without update stanza, all existing allocations are stopped, while new allocations start up, and may cause a service degradation or an outage.  You can regain this disructive behavior by setting `maz_parallel` to 0.

You can do more wordsmitthing if you are up for it?